src/main/java/net/dries007/tfc/client/IngameOverlays.java

```diff
@@ -170,9 +173,58 @@ private static void renderHealthBar(LivingEntity entity, ForgeIngameGui gui, Pos
         gui.left_height += 10;
     }
 
+    public static void renderFamiliarity(RenderLivingEvent.Post<?, ?> event)
```

Comment:
I think this should really be in `ClientForgeEventHandler`, since it's relation to everything else in `IngameOverlays` (everything using the explicit overlay system that is not rendered via events), doesn't really exist.